src/zenml/cli/pipeline.py

```diff
@@ -87,25 +89,165 @@ def register_pipeline(source: str) -> None:
     pipeline_instance.register()
 
 
-@pipeline.command("run", help="Run a pipeline with the given configuration.")
+@pipeline.command("build", help="Build Docker images for a pipeline.")
+@click.argument("pipeline_name_or_id")
+@click.option(
+    "--version",
+    "-v",
+    type=str,
+    required=False,
+)
+@click.option(
+    "--config",
+    "-c",
+    "config_path",
+    type=click.Path(exists=True, dir_okay=False),
+    required=False,
+)
+@click.option(
+    "--stack",
+    "-s",
+    "stack_name_or_id",
+    type=str,
+    required=False,
+)
+@click.option(
+    "--output",
+    "-o",
+    "output_path",
+    type=click.Path(exists=False, dir_okay=False),
+    required=False,
+)
+def build_pipeline(
+    pipeline_name_or_id: str,
+    version: Optional[str] = None,
```

Comment:
Even if the pipeline code/DAG changed since version 1, it would be nice to get a warning explaining why I'm forced to create a build against a different version than the one I explicitly specify with `--version`.